pkg/server/registry/apigroups/acorn/volumes/translator.go

```diff
@@ -22,6 +22,33 @@ type Translator struct {
 }
 
 func (t *Translator) FromPublicName(ctx context.Context, namespace, name string) (string, string, error) {
+
+	i := strings.LastIndex(name, ".")
+	// If there is not a period, parse it not as an alias
+	if i == -1 || i+1 > len(name) {
+		return "", name, nil
+	}
+
+	// parse it of the form <appName>.<shortVolName>
+	prefix := name[:i]
+	volumeName := name[i+1:]
+
+	volumes := &apiv1.VolumeList{}
+	err := t.c.List(ctx, volumes, &kclient.ListOptions{
+		Namespace: namespace,
+		LabelSelector: klabels.SelectorFromSet(map[string]string{
+			labels.AcornAppName:    prefix,
+			labels.AcornVolumeName: volumeName,
+		}),
+	})
+	if err != nil {
+		return "", "", err
+	}
+
+	if len(volumes.Items) == 1 {
+		return "", volumes.Items[0].Name, nil
```

Comment:
It looks like the first string returned from this function is always empty. Can we remove this return value?

Reply:
Could potentially, but didn't want to dig into the debugging and refactoring of this. Doesn't change how it behaved before or add any new fields so I left it as is for now.